src/routes/api.routers.ts

```diff
@@ -5,3 +5,4 @@ export const apiRoutes = express.Router();
 
 apiRoutes.get('/products', apiController.getAllPhones);
 apiRoutes.get('/products/:id', apiController.getPhone);
+apiRoutes.get('/products/:id/recommended', apiController.getPhonesRecommended);
```

Comment:
I think that there shouldn't be :id. You have to return a list of recommended phones with colour filter.